dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramRuleActionController.java

```diff
@@ -86,14 +85,19 @@ public void getDataExpressionDescription( @RequestBody String condition, @Reques
         }
         else
         {
-            message.setDescription(
-                ObjectUtils.firstNonNull( result.getErrorMessage(), result.getException().getMessage() ) );
```

Comment:
Usage of `firstNonNull` would actually result in a NPE since `result.getException()` would be `null`.